interactions/client/models/command.py

```diff
@@ -106,6 +106,7 @@ class Option(DictSerializerMixin):
     :ivar Optional[bool] autocomplete?: A status denoting whether this option is an autocomplete option.
     :ivar Optional[Dict[Union[str, Locale], str]] name_localizations?: The dictionary of localization for the ``name`` field. This enforces the same restrictions as the ``name`` field.
     :ivar Optional[Dict[Union[str, Locale], str]] description_localizations?: The dictionary of localization for the ``description`` field. This enforces the same restrictions as the ``description`` field.
+    :ivar Optional[str] converter: How the argument for the option in your function is passed, if any
```

Comment:
```suggestion
    :ivar Optional[str] converter: How the option value is passed to the function, if different than ``name``
```
I think this makes more sense